### PR TITLE
General fixes and stuff

### LINKED ISSUE #1206: some buffers not being serialised to JSON but written as bytes
**Describe the bug**
Have come across a couple of structures that wkit shows the internal structure of, and which can be written to JSON as individual nodes, but when the whole file is converted to json they are written as a string of bytes.
Behaviour observed on base\environment\decoration\small_shops\stands\mannequin_real\generated\mannequin_real_man_a_doggy.app
trying to access the appearance[0] parametersBuffer
and
base\worlds\03_night_city\sectors\_generated\collisions\03_night_city.geometry_cache
trying to access the bufferTableSectors

**To Reproduce**
1. Add either of the above files to a project
2. Right click and convert to json in the project tree
3. look at the JSON and the mentioned sections are just Byte arrays.
4. Open the file and export the section to JSON and you get the individual sub sections/properties etc.

**Expected behavior**
JSON with the properties visible rather than the byte strings. 

**Version (please complete the following information):**
 - checked with main.

### LINKED ISSUE #1207: LocKeys
TweakDB Browser can't show a single LocKey. When I click on a string with a LocKey on it, the string becomes empty.
LockKey Browser shows two columns. When I type a number in there, nothing happens too.
I don't understand what is wrong. I have the latest .Net framework runtime installed. I believe this have started when Cyberpunk got an update.

### PR DESCRIPTION
**Pull request by seberoth**
**!!! This bumps JSON to v0.0.4 !!!**

Implemented:
- Switch LocKey Browser Primaray column from Text to Numeric. Partially fixes #1207 
- Buffer serialization for `SerializationDeferredDataBuffer` and `SharedDataBuffer`. Fixes #1206 

Fixed:
- NodeRef hash calculation for paths with aliases

Disabled unused graph viewer for scene files. See #1208

### CODE REVIEW

**Comment by Simarilius-uk** [comment]
confirm its fixed 1206 for the app files, but it seems to still not have fixed the BufferTables in the geometry_cache

**Comment by seberoth** [comment]
> confirm its fixed 1206 for the app files, but it seems to still not have fixed the BufferTables in the geometry_cache

Yep, thats "intended" since we have no writer for it yet. So even if you change it in wkit, the changes will not be saved.

File: WolvenKit.RED4/Types/Pools/NodeRefPool.cs [hunk 1]
```diff
@@ -1,14 +1,19 @@
 ﻿using System.Collections.Concurrent;
 using System.Text;
 using WolvenKit.Common.FNV1A;
+using WolvenKit.Core.Interfaces;
 
 namespace WolvenKit.RED4.Types;
 
 public static class NodeRefPool
 {
+    public static ILoggerService? Logger;
```
Comment (rfuzzo) [inline WolvenKit.RED4/Types/Pools/NodeRefPool.cs:10]: is this assigned anywhere?

**Review by rfuzzo — CHANGES_REQUESTED** [review]
I trust you on the json converters, the rest looks good. one comment

File: WolvenKit.RED4/Types/Pools/NodeRefPool.cs (hunk 1 above)
Comment (seberoth) [reply WolvenKit.RED4/Types/Pools/NodeRefPool.cs:10]: No, not yet. Should have commented that out for now...

**Review by rfuzzo — APPROVED** [review]
danke

**Pull request merged**